instrumentation/opentelemetry-instrumentation-confluent-kafka/tests/test_instrumentation.py

```diff
@@ -58,3 +60,14 @@ def test_instrument_api(self) -> None:
 
         consumer = instrumentation.uninstrument_consumer(consumer)
         self.assertEqual(consumer.__class__, Consumer)
+
+    def test_context_setter(self) -> None:
+        context_setter = KafkaContextSetter()
+
+        carrier_dict = {"key1": "val1"}
+        context_setter.set(carrier_dict, "key2", "val2")
+        self.assertGreaterEqual(carrier_dict.items(), {"key2": "val2".encode()}.items())
```

Comment:
After the context_setter.set the original carrier_dict changed?
or you meant to check the context_setter carrier?

Reply:
yes that's what the assert is checking that the original `carrier_dict` has the `key2:val2` pair added on line 68